backend/README.md

```diff
@@ -59,23 +59,37 @@ To update the username or password after it's been set:
 
 This project uses Celery for handling asynchronous execution. Celery tasks are managed through various queues and consumed by workers.
 
-> ETL, TASK, and API Deployment are using these asynchronous workers. Log management also utilizes Celery.
+> ETL, TASK, and API Deployment tasks are handled by these asynchronous workers. Log management also utilizes Celery.
 
 ### Queues
 
-| Queue Name                 | Description                         |
-|----------------------------|-------------------------------------|
-| `celery`                   | Default queue for general Celery tasks. |
-| `celery_periodic_logs`     | Queue for logging history tasks.    |
-| `celery_log_task_queue`    | Queue for persisting logs.          |
-| `celery_api_deployments`   | Queue for API deployment tasks.     |
+| Queue Name                 | Description                                    | Tasks                                                 |
+|----------------------------|------------------------------------------------|-------------------------------------------------------|
+| `celery`                   | Default queue for general Celery tasks, including those without a defined queue. | Webhook notifications, Pipeline (ETL, Tasks) Executions. |
+| `celery_periodic_logs`     | Queue for logging history tasks.               |                                                       |
+| `celery_log_task_queue`    | Queue for persisting logs.                     |                                                       |
```

Comment:
@muhammad-ali-e can you elaborate on these too? Does `celery_log_task_queue` persist to the DB and the other push log messages to FE?

Reply:
@chandrasekharan-zipstack Updated. Thanks for the catch

Reply:
Doc updated.